torch/distributed/distributed_c10d.py

```diff
@@ -2179,6 +2179,8 @@ def scatter(tensor, scatter_list=None, src=0, group=None, async_op=False):
     """
     Scatters a list of tensors to all processes in a group.
 
+    Complex tensors are supported.
```

Comment:
nit: can we make this right before the `Args` section? we don't want to separate the descriptions

Reply:
Sure thing.